python/tests/test_router_microservice.py

```diff
@@ -9,17 +9,24 @@
 
 
 class UserObject(object):
-    def __init__(self, metrics_ok=True):
+    def __init__(self, metrics_ok=True, ret_meta=False):
         self.metrics_ok = metrics_ok
+        self.ret_meta = ret_meta
 
-    def route(self, X, features_names):
+    def route(self, X, features_names, **kwargs):
```

Comment:
Would it make sense to also add a test for `route(...)` methods without the `meta` keyword to ensure backwards compatibility?

Reply:
Isn't that already gappening in the `test_router_ok` test?

Reply:
Added test